authcontroller.py

```diff
@@ -889,8 +889,7 @@ def exportmembers():
 
 
 def generateRow(users):
-    yield "First Name, Last Name, Email, Eppn, ORCID ID \n"
+    yield "Email, Eppn, ORCID ID \n"
```

Comment:
need to remove the spaces surrounding the field names. Otherwise they get included into the name when the file gets imported into spreadsheet app or parsed as CSV file